packages/bpk-docs/src/components/DocsPageBuilder/NeoDocsPageBuilder.js

```diff
@@ -215,7 +215,7 @@ const NeoDocsPageBuilder = props => {
           )
         }
       />
-      <AlternatingPageContent sections={sections} />
+      <AlternatingPageContent sections={sections} invert={context.wrapped} />
```

Comment:
If the page is wrapped, we set `invert` in order to make the light/dark pattern go dark/light instead.